game/locations.rpy

```diff
@@ -160,7 +160,7 @@ init -20 python:
             """
             return self.sunrise <= datetime.datetime.now().time() < self.sunset
 
-        def draw(self, dissolve_all=False, full_redraw=False):
+        def draw(self, dissolve_all=False, full_redraw=False, no_dissolve=False):
```

Comment:
I think there is a forgotten added parameter here

Reply:
Ah, leftover from when I was tinkering before making a new function.